pkgs/os-specific/linux/pflask/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchurl, python
+}:
+
+stdenv.mkDerivation rec {
+  name = "pflask-${version}";
+  version = "git-2015-12-17";
+  rev = "599418bb6453eaa0ccab493f9411f13726c1a636";
+
+  src = fetchurl {
+    url = "https://github.com/ghedo/pflask/archive/${rev}.tar.gz";
+    sha256 = "2545fca37f9da484b46b6fb5e3a9bbba6526a9725189fe4af5227ef6e6fca440";
+  };
+
+  buildInputs = [ python ];
+
+  configurePhase = ''
+    ln -s ${fetchurl {
```

Comment:
possibly, but I don't see any waf package..
(the `mpv` package also fetches it similarly, others bundle waf with the code)